File: tutorias-inf-uva-frontend/src/app/models/tutoria.ts
import { SolicitudTutoria } from "./solicitud-tutoria";

export class Tutoria {

	id?: number;
	cancelada?: boolean;
	motivoCancelacion?: string;
	solicitudTutoria?: SolicitudTutoria;

	constructor(id?: number, cancelada?: boolean, motivoCancelacion?: string, solicitudTutoria?: SolicitudTutoria) {
		if (id) {
			this.id = id;
		}

		if (cancelada) {
			this.cancelada = cancelada;
		}

		if (motivoCancelacion) {
			this.motivoCancelacion = motivoCancelacion;
		}

		if (solicitudTutoria) {
			this.solicitudTutoria = solicitudTutoria;
		}
	}
}

File: tutorias-inf-uva-api/src/main/java/es/uva/inf/tutorias/business/domain/models/Tutoria.java
package es.uva.inf.tutorias.business.domain.models;

import java.io.Serializable;

public class Tutoria implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long id;
	private Boolean cancelada;
	private String motivoCancelacion;
	private SolicitudTutoria solicitudTutoria;

	public Tutoria() {
	}

	public Long getId() {
		return this.id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Boolean getCancelada() {
		return this.cancelada;
	}

	public void setCancelada(Boolean cancelada) {
		this.cancelada = cancelada;
	}

	public String getMotivoCancelacion() {
		return this.motivoCancelacion;
	}

	public void setMotivoCancelacion(String motivoCancelacion) {
		this.motivoCancelacion = motivoCancelacion;
	}

	public SolicitudTutoria getSolicitudTutoria() {
		return solicitudTutoria;
	}

	public void setSolicitudTutoria(SolicitudTutoria solicitudTutoria) {
		this.solicitudTutoria = solicitudTutoria;
	}

}